bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -581,6 +581,8 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       o <- p.nonEmptyObservations
       if o.band == Band.BAND_3 && (p.proposalClass match {
         case q: QueueProposalClass if q.band3request.isDefined => false
+        case f: QueueProposalClass if f.band3request.isDefined => false
```

Comment:
This looks like it is there twice?  Did you mean to change it to some other proposal class?

Reply:
ups